src/test/App.svelte.test.ts

```diff
@@ -1,96 +1,113 @@
-import { afterEach, beforeEach, describe, expect, type MockInstance, test, vi } from "vitest";
-import { render, type RenderResult, screen, waitFor } from "@testing-library/svelte";
-import App from "../App.svelte";
-import { mockIPC, mockWindows } from "@tauri-apps/api/mocks";
-import { authenticationResponseMock, envVariablesMock, fileEntryListMock } from "./mock-data.mock";
-import { settings } from "../lib/util/settings";
-import { awaitNthTicks } from "./util/util";
-
+import { afterEach, beforeEach, describe, expect, type MockInstance, test, vi } from 'vitest';
+import { render, type RenderResult, screen, waitFor } from '@testing-library/svelte';
+import App from '../App.svelte';
+import { mockIPC, mockWindows } from '@tauri-apps/api/mocks';
+import { authenticationResponseMock, fileEntryListMock, secretVariablesMock } from './mock-data.mock';
+import { settings } from '../lib/util/settings';
+import { awaitNthTicks } from './util/util';
 
 describe('App.svelte', () => {
-    let container: RenderResult<App>
-    let setAuthResponseSpy: MockInstance<[Promise<AuthenticationResponse | null>], void>
+    let container: RenderResult<App>;
+    let setAuthResponseSpy: MockInstance<
+    [Promise<AuthenticationResponse | null>],
+    void
+  >;
```

Comment:
```suggestion
    let setAuthResponseSpy: MockInstance<
        [Promise<AuthenticationResponse | null>],
        void
    >;
```
Er linteren litt overivrig?